beanie/odm/fields.py

```diff
@@ -363,10 +363,10 @@ def repack_links(
 
     @classmethod
     async def fetch_many(cls, links: List[Link]):
-        coros = []
-        for link in links:
-            coros.append(link.fetch())
-        return await asyncio.gather(*coros)
```

Comment:
This method used to return here previously. I don't see how are the "results" being returned now?

Reply:
good catch. will add test to his this is breaking change